docs/source/user/notebook.md

```diff
@@ -0,0 +1,217 @@
+   <iframe src="https://www.youtube-nocookie.com/embed/Asa_ML45HP8?rel=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+```
+
+(tab-completion)=
+
+Tab completion (activated with the `Tab` key) can now include additional
+information about the types of the matched items:
+
+```{raw} html
+<div class="jp-youtube-video">
+   <iframe src="https://www.youtube-nocookie.com/embed/MuNr0i8LgpM?rel=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+```
+
+Note: IPython 6.3.1 has temporarily disabled type annotations.
+To re-enable them, add `c.Completer.use_jedi = True` to an
+[ipython_config.py](https://ipython.readthedocs.io/en/stable/development/config.html) file.
+
+(user-tooltip)=
+
+The tooltip (activated with `Shift Tab`) contains additional
+information about objects:
+
+```{raw} html
+<div class="jp-youtube-video">
+   <iframe src="https://www.youtube-nocookie.com/embed/TgqMK1SG7XI?rel=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+```
+
+(create-console)=
+
+You can connect a {ref}`code console <code-console>` to a notebook kernel to have a log of
+computations done in the kernel, in the order in which they were done.
+The attached code console also provides a place to interactively inspect
+kernel state without changing the notebook. Right-click on a notebook
+and select “New Console for Notebook”:
+
+```{raw} html
+<div class="jp-youtube-video">
+   <iframe src="https://www.youtube-nocookie.com/embed/eq1l7DBngQQ?rel=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+```
+
+(kernel-history)=
+
+You can iterate through the kernel history in a document cell using `Alt Up-Arrow` and `Alt Down-Arrow`. To use this feature, enable kernel history access in the notebook settings.
+
+(cell-toolbar)=
+
+## Cell Toolbar
+
+In each cell, there is a toolbar that provides quick access to
+commonly-used functions, if there is enough room to display it on the first line.
+
+If you would like to disable the cell toolbar, open the Settings Editor, choose the
+Cell Toolbar in the left panel, and uncheck "Show cell toolbar".
+
+Administrators can turn off the cell toolbar by disabling its extension, by running:
+
+```bash
+jupyter labextension disable @jupyterlab/cell-toolbar-extension
+```
+
+on the command line. Administrators can turn it back on by running:
+
+```bash
+jupyter labextension enable @jupyterlab/cell-toolbar-extension
+```
+
+(notebook-trust)=
+
+## Trust
+
+JavaScript and HTML in notebooks created on other machines are not trusted,
+which results in sanitization of HTML and interactive outputs not being
+displayed until the notebook is explicitly trusted.
+
+The trust status of the active notebook is indicated by a shield icon in the
+status bar; a checkmark ({{ trusted }}) in the shield indicates a trusted
```

Comment:
This substitution is not rendering properly because I think you need to add `substitution` to the list of enabled myst extensions in `conf.py`